code/src/components/TodoList.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import moment from 'moment'
+import styled from 'styled-components'
+import { useSelector, useDispatch } from 'react-redux'
+
+import { todos } from '../reducers/todos' 
+
+export const TodoList = () => {
+  const dispatch = useDispatch();
+  
+  const items = useSelector(store => store.todos.items);
+
+  const onClickDelete = id => {
+    dispatch(todos.actions.removeItem(id))
+  }
+
+  const onChecked = id => {
+    dispatch(todos.actions.toggleComplete(id))
+  }
+  
+  return (
+    <>
+            {items.map(item => (
+        <TodoContainer key={item.id}>
+          <List>
+      <label>
+          <input type="checkbox"
+          checked={item.complete}
+          onChange={() => onChecked(item.id)}
+          />
+      </label>
+  </List>
```

Comment:
A bit of indentation that needs fixing to make the code just a bit more readable 😊